app/models/katello/content_view_docker_filter_rule.rb

```diff
@@ -0,0 +1,23 @@
+module Katello
+  class ContentViewDockerFilterRule < Katello::Model
+    self.include_root_in_json = false
+
+    belongs_to :filter,
+               :class_name => "Katello::ContentViewDockerFilter",
+               :inverse_of => :docker_rules,
+               :foreign_key => :content_view_filter_id
+
+    validates_lengths_from_database
```

Comment:
I would like to allow filters for non-existent tags. For example, disallowing "latest" is something asked for so even if it doesn't exist currently, it may sneak in in the future in the repo and I'd want to pre-emptively exclude it.